packages/office-ui-fabric-react/src/components/Stack/examples/Stack.Horizontal.Basic.Example.tsx

```diff
@@ -1,43 +1,42 @@
 import * as React from 'react';
-import { Stack } from '../Stack';
-import { IStackStyles, IStackTokens } from '../Stack.types';
-import { IStackItemStyles } from '../StackItem/StackItem.types';
-import { DefaultPalette } from 'office-ui-fabric-react/lib/Styling';
+import { DefaultPalette, Stack, IStackStyles, IStackTokens, IStackItemStyles } from 'office-ui-fabric-react';
 
-export class HorizontalStackBasicExample extends React.Component<{}, {}> {
-  public render(): JSX.Element {
-    const stackStyles: IStackStyles = {
-      root: {
-        background: DefaultPalette.themeTertiary
-      }
-    };
-    const stackItemStyles: IStackItemStyles = {
-      root: {
-        background: DefaultPalette.themePrimary,
-        color: DefaultPalette.white,
-        padding: 5
-      }
-    };
-    const itemAlignmentStackStyles: IStackStyles = {
-      root: {
-        background: DefaultPalette.themeTertiary,
-        height: 100
-      }
-    };
+// Styles definition
+const stackStyles: IStackStyles = {
+  root: {
+    background: DefaultPalette.themeTertiary
+  }
+};
+const stackItemStyles: IStackItemStyles = {
+  root: {
+    background: DefaultPalette.themePrimary,
+    color: DefaultPalette.white,
+    padding: 5
+  }
+};
+const itemAlignmentsStackStyles: IStackStyles = {
```

Comment:
nit: `itemAlignmentsStackStyles` is different from `itemAlignmentStackTokens` later in this file and `itemAlignmentStackStyles` in later files - rename this to `itemAlignmentStackStyles`?